synth.py

```diff
@@ -70,12 +73,12 @@ def generate_data_api(gapic):
   ]
 
   package_name = f'com.google.bigtable.v2'
-  java.fix_proto_headers(library / f'proto-google-cloud-bigtable-v2')
-  java.fix_grpc_headers(library / f'grpc-google-cloud-bigtable-v2', package_name)
+  java.fix_proto_headers(library / f'proto-google-cloud-bigtable-v2-java')
```

Comment:
I don't think these should be changing. This should have generated at least the changes found here: https://github.com/googleapis/java-bigtable/pull/222 which we have yet to merge in.

Reply:
Sorry, that was silly me. I updated the synth.py, but did not update my command to call synthtool, so I just generated java-gerafeas (the repo I was migrating before java-bigtable).